scripts-5.8/postinstall.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+export PKG_PATH="$MIRROR/pub/OpenBSD/`uname -r`/packages/`arch -s`/"
+
+# create vagrant user/group and home directory
+groupadd vagrant
+adduser -noconfig -unencrypted -batch vagrant vagrant,wheel vagrant vagrant
```

Comment:
Is this the only backwards incompatible change? What if we check if the `vagrant` user exists before trying to create it?